cli/emit.rs

```diff
@@ -206,17 +208,15 @@ fn get_tsc_roots(
     .into_iter()
```

Comment:
This `get_tsc_roots` being in "emit" doesn't make much sense anymore because tsc never emits anymore. So this function is using `has_ts_check` in order to completely skip parsing with swc when type checking if module analysis has already occurred once before.

Reply:
Makes sense to me. Open a follow up issue?

Reply:
Opened https://github.com/denoland/deno/issues/15535